tools/Google.Cloud.Tools.VersionCompat/Google.Cloud.Tools.VersionCompat.csproj

```diff
@@ -11,6 +10,7 @@
     <PackageReference Include="Mono.Cecil" Version="0.10.3" />
     <PackageReference Include="NuGet.Client" Version="4.2.0" />
```

Comment:
Not related to this PR, but is `NuGet.Client` required? The code that loads from nuget doesn't appear to use this?

Reply:
Ooh, good question. Will try it without, and remove in this PR if it's not necessary.